Tests/TABResourceLoaderTests/Services/NetworkDataResourceServiceTests.swift

```diff
@@ -177,5 +177,10 @@ class NetworkDataResourceServiceTests: XCTestCase {
       mockSession.capturedCompletion!(nil, nil, nil)
     }
   }
-
+  
+  func testSessionIsInvalidatedOnDeinit() {
```

Comment:
Small thing but doesn't follow your convention of `test_` elsewhere. Would be nice to keep it consistent. I've started doing it this way recently too, helps keep it readable.